drivers/mtd/w25qxxxjv.c

```diff
@@ -594,6 +601,32 @@ static void w25qxxxjv_write_disable(FAR struct w25qxxxjv_dev_s *priv)
   while ((status & STATUS_WEL_MASK) != STATUS_WEL_DISABLED);
 }
 
+/****************************************************************************
+ * Name:  w25qxxxjv_quad_enable
+ ****************************************************************************/
+
+static void w25qxxxjv_quad_enable(FAR struct w25qxxxjv_dev_s *priv)
+{
+  w25qxxxjv_command_read(priv->qspi, W25QXXXJV_READ_STATUS_2,
+                         (FAR void *)priv->cmdbuf, 1);
+
+  while ((priv->cmdbuf[0] & STATUS2_QE_MASK) != STATUS2_QE_ENABLED)
```

Comment:
Why does this need to be looping?

Reply:
You are right, there is no need to be looping.  I was imitating the w25qxxxjv_write_enable/write_disable functions.  I will remove the loop.